error-analyzer/src/main/java/com/google/sps/servlets/HelloInLog4J2.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/*
+USE:
+Import Logger and LogManager in all java files and create a logger object.
+Use logger.*appropriate level*("log msg").
+
+Log statements appear both in console and target/logs/app.log
+If your log file reaches 5MB in size, it gets saved in YYYY-MM directory inside target/logs
+The new log data is stored in app.log
+
+The log data is appended (not overwritten) in app.log .
```

Comment:
This is multi-line comment and not javadoc.
https://en.wikipedia.org/wiki/Javadoc